packages/services/src/view-service/asset-metadata-by-id.ts

```diff
@@ -16,7 +17,10 @@ export const assetMetadataById: Impl['assetMetadataById'] = async ({ assetId },
   if (localMetadata) return { denomMetadata: localMetadata };
 
   const remoteMetadata = await querier.shieldedPool.assetMetadataById(assetId);
-  if (remoteMetadata) {
+
+  const isIbcAsset = remoteMetadata && assetPatterns.ibc.matches(remoteMetadata.display);
+
+  if (remoteMetadata && !isIbcAsset) {
```

Comment:
It turns out that in addition to the block processor, we also request metadata from the node in the `assetMetadataById` rpc. 
This doesn't seem right to me, but I will create a separate issue for this, since this PR is blocking the release